packages/unlock-js/src/walletService.ts

```diff
@@ -14,6 +16,47 @@ interface CreateLockOptions {
   creator?: string
 }
 
+export const UnlockSwapPurchaserABI = [
+  `function swapAndCall(address lock, address srcToken, uint amountInMax,address uniswapRouter, bytes memory swapCalldata, bytes memory callData) public payable returns(bytes memory)`,
+]
```

Comment:
Let's move to a specific file like we do for ERC20 ABI or the Key Manager contract ABI for calrity